src/drawDirective.coffee

```diff
@@ -24,7 +24,7 @@ angular.module('ui-leaflet')
     restrict: 'A'
     scope: false
     replace: false
-    require: ['leaflet']
+    require: ['^^leaflet']
```

Comment:
https://docs.angularjs.org/guide/directive#creating-directives-that-communicate

> The ^^ prefix means that this directive searches for the controller on its parents. (A ^ prefix would make the directive look for the controller on its own element or its parents; without any prefix, the directive would look on its own element only.)

So if this makes it work only by searching on it's parent then I am opposed (can be swayed). If it can work with both then I see no issue. 

This is actually interesting because in reality I think this directive only depends on leaflet directive to get its relative [map being loaded](https://github.com/angular-ui/ui-leaflet-draw/blob/master/src/drawDirective.coffee#L37) . 

Anyway, if we move this to nesting I guess this would be a minor change or major? What do you think? 

On breaking paradigm shift is making this nested as every attribute  in ui-leaflet 1.X is on the same directive element.

Anyway, if travis didn't build the dist file this PR will need to do so. As I want to see if it still passes .